src/main/java/me/zeroeightsix/kami/module/modules/misc/AntiAFK.kt

```diff
@@ -150,19 +150,13 @@ object AntiAFK : Module() {
         TURN(turn)
     }
 
-    private val isBaritoneActive: Boolean = if (mc.player != null) BaritoneAPI.getProvider().primaryBaritone.customGoalProcess.isActive else false
-
     private fun baritoneGotoXZ(x: Int, z: Int) {
         BaritoneAPI.getProvider().primaryBaritone.customGoalProcess.setGoalAndPath(GoalXZ(x, z))
     }
 
-    private fun baritoneCancel() {
-        BaritoneAPI.getProvider().primaryBaritone.pathingBehavior.cancelEverything()
-    }
-
     init {
         walk.settingListener = Setting.SettingListeners {
-            if (isBaritoneActive) BaritoneAPI.getProvider().primaryBaritone.pathingBehavior.cancelEverything()
+            BaritoneAPI.getProvider().primaryBaritone.pathingBehavior.cancelEverything()
```

Comment:
I think this listener is not actually doing anything...I'm going to take a look

Reply:
Yes it is. It's run when the setting is changed.